src/App.js

```diff
@@ -14,16 +15,12 @@ class App extends Component {
   onChangeInputHandler = event => this.setState({value: event.target.value})
 
   render() {
-    const { tasks, value } = this.state
+    const { tasks } = this.state
 
     return (
       <div className="app">
         <Title text='ToDo App!!!' tag='h3'/>
-        <div className="app__input">
-          <label>Input task:</label>
-          <input onChange={this.onChangeInputHandler} value={value}></input>
-          <button onClick={this.addTask}>Add</button>
-        </div>
+        <Input value={this.state.value} onChangeHandler={this.onChangeInputHandler.bind()} onClickHandler={this.addTask} />
```

Comment:
why value is not destructured anymore? 
binding methods inline is very inefficient. On each render, engine has to bind method. Please, make a move and bind this method in the constructor. 
I assume, that addTask should be also binded. 
Later, we will learn how to avoid this stuff ;P